src/liballoc/vec.rs

```diff
@@ -780,21 +816,11 @@ impl<T> Vec<T> {
     /// }
     /// ```
     ///
-    /// In this example, the vector gets expanded from zero to four items
-    /// without any memory allocations occurring, resulting in vector
-    /// values of unallocated memory:
-    ///
-    /// ```
-    /// let mut vec: Vec<char> = Vec::new();
-    ///
-    /// unsafe {
-    ///     vec.set_len(4);
-    /// }
-    /// ```
+    /// (Instead, one would normally use [`clear`] in this situation.)
```

Comment:
No need for a parenthetical imo; it's important info.

Reply:
Perhaps it should be made more clear that `clear` differs from `set_len` exactly in how they behave wrt dropping the contents?